vsintegration/src/FSharp.Editor/Hints/NativeToRoslynHintConverter.fs

```diff
@@ -21,7 +25,14 @@ module NativeToRoslynHintConverter =
         let text = taggedText.Text
         RoslynTaggedText(tag, text)
 
+    let nativeToRoslynFunc nativeFunc =
+        Func<Document, CancellationToken, Task<ImmutableArray<RoslynTaggedText>>>(fun doc ct ->
+            nativeFunc doc ct
+            |> Async.map (List.map nativeToRoslynText >> ImmutableArray.CreateRange)
+            |> Async.StartAsTask)
```

Comment:
We should probably just pass the `ct` here instead of the `nativeFunc` which should then get it automatically via `async`.

Reply:
Yep, thanks, there you gooo.